db/knex_migrations/2024-08-24-0000-conditions.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function (knex) {
+    return knex.schema
+        .alterTable("monitor", function (table) {
+            table.text("conditions").notNullable().defaultTo("[]");
```

Comment:
I am just testing it on MariaDB, and found a strange bug of this pr.

When I click login button, it doesn't work. In the console, it throws an exception: 

> Trace: SyntaxError: Unexpected end of JSON input
>    at JSON.parse (<anonymous>)
>  at Proxy.toJSON (C:\Users\Louis-K6\Dropbox\My Projects\uptime-kuma\server\model\monitor.js:159:30)

Line: 159
>        conditions: JSON.parse(this.conditions),


And then, I go into MariaDB and saw that is `No default`, while in my SQLite, it is `[]`. Any ideas?

![image](https://github.com/user-attachments/assets/ca677b57-220d-46be-87ac-051be41f9664)

Reply:
Omg... I just realized that `accepted_statuscodes_json` also have the same issue... 

Wait... What the hell: https://github.com/knex/knex/issues/2649

Reply:
Wanna cry, it don't have any issues using a raw sql..... 

```
ALTER TABLE `monitor`
	ADD COLUMN `condition_test` TEXT NOT NULL DEFAULT '[]' AFTER `rabbitmq_password`;
```

Reply:
Fixed: #5261